exercises/allergies/example.cpp

```diff
@@ -0,0 +1,46 @@
+#include "allergies.h"
+
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+namespace allergies
+{
+
+allergy_test::allergy_test(const int test_result)
```

Comment:
The `const` in this function definition doesn't have any real effect since it's passed by value instead of reference. It works fine, but that's not usually the convention.